tests/functional/pv/pv_services/test_selinux_with_rwop.py

```diff
@@ -0,0 +1,362 @@
+import logging
+import pytest
+import yaml
+import random
+import time
+import tempfile
+from datetime import datetime
+
+from ocs_ci.helpers import helpers
+from ocs_ci.helpers.helpers import verify_log_exist_in_pods_logs
+from ocs_ci.ocs import ocp, constants
+from ocs_ci.framework.testlib import E2ETest
+from ocs_ci.ocs.exceptions import (
+    PodNotCreated,
+    CommandFailed,
+    ResourceWrongStatusException,
+)
+from ocs_ci.ocs.node import wait_for_nodes_status
+from ocs_ci.ocs.resources import pod as res_pod
+from ocs_ci.ocs.resources.pod import (
+    get_plugin_pods,
+    get_pod_node,
+    wait_for_pods_to_be_running,
+)
+from ocs_ci.utility.retry import retry
+from ocs_ci.utility.utils import run_cmd, TimeoutSampler
+from ocs_ci.utility.templating import dump_data_to_temp_yaml
+from ocs_ci.framework.pytest_customization.marks import (
+    tier1,
+    magenta_squad,
+)
+
+log = logging.getLogger(__name__)
+
+
+@magenta_squad
+@tier1
+class TestSelinuxrelabel(E2ETest):
+    def create_deploymentconfig_pod(self, pvc_name=None, **kwargs):
```

Comment:
can we use create_dc_pods from https://github.com/red-hat-storage/ocs-ci/ests/fixtures.py?